### PR TITLE
Fix duplicate definition of RecipientKeyIdentifier

### PR DESCRIPTION
**Pull request by wbond**
Potential fix for #243 that needs a test

### CODE REVIEW

File: asn1crypto/cms.py [hunk 1]
```diff
@@ -729,6 +729,10 @@ class RecipientKeyIdentifier(Sequence):
         ('other', OtherKeyAttribute, {'optional': True}),
     ]
 
+    def _setup(self):
+        super(RecipientKeyIdentifier, self)._setup()
```
Comment (commonism) [inline asn1crypto/cms.py:733]: `super()._setup()` should be fine

Comment (wbond) [reply asn1crypto/cms.py:733]: This project targets Python 2.6+ currently, hence the parameters passed to super.

Comment (commonism) [reply asn1crypto/cms.py:733]: Unexpected as 2.6 has been EOL for 9, 2.7 for 3 years, but you'll have your reasons.

Comment (wbond) [reply asn1crypto/cms.py:733]: Yeah, I’m more interested in tests for the code you added

**Pull request merged**